src/pages/_app.tsx

```diff
@@ -1,6 +1,10 @@
-import '@/styles/globals.css'
+// import '@/styles/globals.css'
 import type { AppProps } from 'next/app'
-
+import {DSProvider} from '@nypl/design-system-react-components'
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return ( 
```

Comment:
This might've gotten lost somewhere but can you add the following:
```
<Head>
  <meta name="viewport" content="width=device-width, initial-scale=1" />
</Head>
```

You'd need to import `Head` from `next/head` and you can place it above the DSProvider:
```
<>
  <Head>
    <meta name="viewport" content="width=device-width, initial-scale=1" />
  </Head>
  <DSProvider>
    ...
  </DSProvider>
</>
```